data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -38,34 +38,31 @@ def get_survey_question_details(access_token: str, survey_id: str) -> list:
 
 
 def get_no_answer_options_for_question_json() -> dict:
+    """Added for consistency of the schema in BQ"""
     return {
-        "id": None,
-        "text": None,
-        "type": None,
-        "weight": None
+        "id": "",
+        "text": "",
+        "type": "",
+        "weight": ""
     }
 
 
 def parse_answer_options_in_question_answer_json(
-    question_answer_response_json: dict,
-    option_key: str
+    question_answer_response_json: dict
 ) -> dict:
-    if isinstance(question_answer_response_json[option_key], list):
-        result_list = []
-        for answer_option in question_answer_response_json[option_key]:
-            result_list.append({
-                "id": answer_option.get("id"),
-                "text": answer_option.get("text"),
-                "type": answer_option.get("type"),
-                "weight": answer_option.get("weight")
-            })
-        return result_list
-    return [{
-        "id": question_answer_response_json[option_key].get("id"),
-        "text": question_answer_response_json[option_key].get("text"),
-        "type": question_answer_response_json[option_key].get("type"),
-        "weight": question_answer_response_json[option_key].get("weight")
-    }]
+    if isinstance(question_answer_response_json, list):
+        answer_options = question_answer_response_json
+    else:
+        answer_options = [question_answer_response_json]
+    return [
+        {
+            "id": str(answer_option.get("id", "")),
```

Comment:
Is `id` actually optional?

Reply:
If the answer is just text in that case there is no id for it.

Reply:
That is something that would be great to capture in a test.
e.g. you could test `parse_answer_options_in_question_answer_json` specifically and have a test along the lines of `test_should_not_return_id_for_text_only_response` (or something like that).